CHANGELOG.md

```diff
@@ -1,21 +1,24 @@
 ---
-
-MOVE THIS SECTION TO THE CORRECT LOCATION
-
-### Misc. Updates
-
-- Model updates for the following services
----AWSDynamoDB
 # AWS Mobile SDK for iOS CHANGELOG
 
 ## Unreleased
 
 -Features for next release
 
+## 2.19.1
```

Comment:
doesn't this get populated automatically?

Reply:
Oh does it, I thought it was just for amplify-ios.

Reply:
Updated